lib/passports.js

```diff
@@ -101,14 +101,26 @@ module.exports = {
                 passport.tokens = JSON.stringify(query.tokens)
               }
 
-              // Save any updates to the Passport before moving on
-              app.services.FootprintService.update('Passport', passport.id, passport) .then(passport => {
-                app.services.FootprintService.find('user', passport[0].user.id)
-                  .then(user => {
-                    next(null, user)
-                  })
-                  .catch(next)
-              }).catch(next)
+             // TODO: Must be tested for check if ok for both Sequelize and Waterline
+             // Save any updates to the Passport before moving on
+              app.services.FootprintService.find('Passport', passport.id)
+                .then(passport => {
+                  // I can't rely on the update results because on trailpack-sequelize return an array with the updated rows counter
+                  // So, first I read passport model
+                  app.services.FootprintService.update("Passport", passport.id, passport)
+                    .then(() => {
+                      // I read UserId field (very bad) 
+                      return app.services.FootprintService.find('user', passport.UserId)
```

Comment:
`passport.UserId` will not work for waterline and sequelize, don't remember witch one is failing on this